scripting/cwx/item_config.sp

```diff
@@ -305,10 +305,53 @@ int EquipCustomItem(int client, const CustomItemDefinition item) {
 			SetEntProp(itemEntity, Prop_Send, "m_iAccountID", accountid);
 		}
 	}
-	
-	// TODO: implement a version that nullifies runtime attributes to their defaults
-	SetEntProp(itemEntity, Prop_Send, "m_bOnlyIterateItemViewAttributes",
-			!item.bKeepStaticAttributes);
+
+	if(!item.bKeepStaticAttributes) {
+		TF2Attrib_RemoveAll(itemEntity);
+
+		char valueType[2];
+		char valueFormat[64];
+
+		int staticAttribs[16];
+		float staticAttribsValues[16];
+
+		TF2Attrib_GetStaticAttribs(item.defindex, staticAttribs, staticAttribsValues);
```

Comment:
`TF2Econ_GetItemStaticAttributes` instead?  The game can technically store more than that.

Reply:
That's the original approach I used since at the time `TF2Attrib_GetStaticAttribs` didn't work for some reason. I don't think it should cause any issues since I don't think there is a weapon that has over 16 attributes (?)

Reply:
Yeah, that's probably fine.  `TF2Attrib_GetStaticAttribs` returns the attribute count so it might be good to iterate up to that count as long as it's less than 16.

Reply:
Addressed this in the newest commit.